### ISSUE TITLE
Token picker header has insufficient contrast with background

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3790]
### Describe the issue

1. Open designer standalone (nx serve).
2. Open token picker
3. Observe the headers are a very light color which does not meet MAS 1.4.3 success criterion.

**Actual:**
Contrast ratio is in the region of ~1.1:1.

**Expected:**
Contrast ratio of 4.5:1

### Are you using new designer or old designer

New Designer (With action panels)

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/de828746-5c7a-4ecf-9a90-bf9644fe3d5b)

### Browser

Chrome

### Additional context

Related ADO bug: https://msazure.visualstudio.com/One/_workitems/edit/10990979

**Comment by Eric-B-Wu** [comment]
should this be required to meet a contrast ratio? This isn't a contrast between text and its background.

**Comment by Ram-G** [comment]
You're right. I see that the contrast has been fixed for the text recently, and the accessibility requirement is not for non-text colors.

I verified that the accessibility requirement looks satisfied in currently deployed bits.
![image](https://github.com/Azure/LogicAppsUX/assets/10837129/7b23320d-d15b-47e0-9264-823d62ebe7ae)

I've sent the internal ADO back to the testing team to review and close. Let me keep this github issue open until they verify the fix, then I'll go ahead and close this issue.

**Comment by Ram-G** [comment]
The testing team has verified this issue is fixed. So I'm closing out this issue.

For more information please reference the linked ADO work item in the original post.
